website/docs/r/stream_analytics_output_servicebus_topic.html.markdown

```diff
@@ -73,6 +73,10 @@ The following arguments are supported:
 
 * `property_columns` - (Optional) A list of property columns to add to the Service Bus Topic output.
 
+* `system_property_columns` - (Optional) A key-value pair of system property columns for the Service Bus Topic Output.
```

Comment:
Can we elaborate on what these system property columns are used for? Docs mention that they are attached to every outgoing message so perhaps rewording would be helpful.
```suggestion
* `system_property_columns` - (Optional) A key-value pair of system property columns that will be attached to the outgoing messages for the Service Bus Topic Output.
```